Documentation/coding-guidelines/breaking-changes.md

```diff
@@ -16,47 +22,86 @@ To help triage breaking changes, we classify them in to four buckets:
 3. Unlikely Grey Area
 4. Clearly Non-Public
 
-### Bucket 1: Public Contract
+## Bucket 1: Public Contract
 *Clear violation of public contract.*
 
 Examples:
-* throwing a new/different exception type in an existing common scenario
-* An exception is no longer thrown
-* A different behavior is observed after the change for an input
 * renaming a public type, member, or parameter
-* decreasing the range of accepted values within a given parameter
-* A new instance field is added to a type (impacts serialization)
 * changing the value of a public constant or enum member
```

Comment:
or any other breaking change to the API surface area, such as making something sealed, abstract, or removing virtual, removing a base class or interface implementation, or changing a return type other than to a derived type